plugin/src/main/kotlin/io/github/irgaly/gradle/rur/RemoveUnusedResourcesPlugin.kt

```diff
@@ -33,8 +35,8 @@ class RemoveUnusedResourcesPlugin : Plugin<Project> {
         val hasOverrideLintOptions =
             (onlyUnusedResources || disableLintConfig || (overrideLintConfig != null))
         if (hasOverrideLintOptions) {
-            target.allprojects.forEach { project ->
-                project.afterEvaluate {
+            target.rootProject.allprojects.forEach { project ->
+                project.afterEvaluateOrExecute {
```

Comment:
this time, rootProject or some project is finish to evaluate, and cannot call afterEvaluate()